stories/chip.stories.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react'
+
+import { Chip } from '../src'
+import { Add, Close } from '../src/icons'
```

Comment:
instead of `Add` the `Info` icon is more useful. Chips are added via input boxes so we don't need the Add icon here
![image](https://user-images.githubusercontent.com/33563070/125996490-8fd44a09-24f6-4e4d-a7a4-edac1955a6cf.png)

how to add/select chips: 
![image](https://user-images.githubusercontent.com/33563070/125996564-7f993fab-348e-48d6-b474-ceb7a5b54d42.png)